From 59e7a0018c597b91c15eadaaf61ebe99140adff1 Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Wed, 17 Apr 2019 12:25:24 -0700
Subject: [PATCH] TRITON-1414 commit 252de200 broke sdc-image-sync Reviewed by:
 Rob Johnston <rob.johnston@joyent.com> Approved by: John Levon
 <john.levon@joyent.com>

---
 tools/bin/sdc-image-sync | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/bin/sdc-image-sync b/tools/bin/sdc-image-sync
index a44d5de2..1a00c294 100755
--- a/tools/bin/sdc-image-sync
+++ b/tools/bin/sdc-image-sync
@@ -66,7 +66,7 @@ function add_pkg()
 	    else
 		print $0
         }' $usbmnt/config.inc/generic > $usbmnt/config.inc/generic.new
-	mv $usbmnt/config.inc/generic.new $usbmnt /config.inc/generic
+	mv $usbmnt/config.inc/generic.new $usbmnt/config.inc/generic
 }
 
 function update_sdc_zone_pkgs()
@@ -98,7 +98,7 @@ function save_root_password() {
     echo "Updating root password in the config file"
 
     sed -e "s|^root_shadow=.*$|root_shadow='${enc_password}'|" \
-      $usbmt/config > $usbmnt/config.new \
+      $usbmnt/config > $usbmnt/config.new \
       && mv $usbmnt/config.new $usbmnt/config
 }
 
-- 
2.21.0

